app/editor/src/components/tool-bar/ToolBarSection2.tsx

```diff
@@ -0,0 +1,21 @@
+import { Row } from 'tno-core';
```

Comment:
Maybe rename this to ContentStatusSection.tsx and move it to [tno](https://github.com/bcgov/tno)/[app](https://github.com/bcgov/tno/tree/dev/app)/[editor](https://github.com/bcgov/tno/tree/dev/app/editor)/[src](https://github.com/bcgov/tno/tree/dev/app/editor/src)/[features](https://github.com/bcgov/tno/tree/dev/app/editor/src/features)/[content](https://github.com/bcgov/tno/tree/dev/app/editor/src/features/content)/[tool-bar](https://github.com/bcgov/tno/tree/dev/app/editor/src/features/content/tool-bar)/sections/

Reply:
I don't think there is any value in having another generic component.  The current `ToolBarSection` component does most of this.  Just update it so that it can have an optional `icon` property.  Make a specific implementation for `ContentStatusSection` that just shows the status.  You can then override the styling locally to make the component look correct.

`/app/editor/src/features/content/tool-bar/ContentStatusSection.tsx`

```
export const ContentStatusSection: React.FC<IContentStatusSectionProps> = () => {
const { values } = useFormikContext<IContentForm>();
return (<styled.ContentStatusSection>
 <ToolBarSection label="Content Details">{values.status}</ToolBarSection/>
 </styled.ContentStatusSection>
 );
 }